src/olympia/activity/utils.py

```diff
@@ -63,55 +67,74 @@ def _extra_email_reply_parse(self, email):
             return split_email[0]
 
     def get_uuid(self):
-        for to in self.email.get('To', []):
+        to_header = self.email.get('To', [])
+        for to in to_header:
             address = to.get('EmailAddress', '')
             if address.startswith(self.address_prefix):
                 # Strip everything between "reviewreply+" and the "@" sign.
                 return address[len(self.address_prefix):].split('@')[0]
         log.exception(
             'TO: address missing or not related to activity emails. (%s)'
-            % self.email)
+            % to_header)
         raise ActivityEmailUUIDError(
-            'TO: address doesn\'t contain activity email uuid (%s)'
-            % self.email)
+            'TO: address doesn\'t contain activity email uuid (%s).'
+            % to_header)
 
-    def get_body(self):
-        return self.reply
 
+def add_email_to_activity_log_wrapper(message):
+    note = None
+    reason = 'Undefined Error.'
+    try:
+        parser = ActivityEmailParser(message)
+        note = add_email_to_activity_log(parser)
+    except ActivityEmailError as exception:
+        reason = str(exception)
+
+    if not note:
+        try:
+            bounce_mail(message, reason)
+        except:
```

Comment:
I think KeyError is probably the worst we'll get tbh, I just didn't want further exceptions to be thrown in something that is effectively handling an exception.